src/multiple_shooting_objective.jl

```diff
@@ -0,0 +1,50 @@
+export MultipleShootingObjective, multiple_shooting_objective
+
+immutable MultipleShootingObjective{F,C} <: Function
+  multiple_shooting_cost_function::F
+  constraints::C
+end
+
+my_problem_new_parameters = function (prob::ODEProblem,p,t1,t2,data)
```

Comment:
somehow the DiffEqBase.jl one should be extended with `tspan` and `data` as default args I think. Look into this later.